### PR TITLE
Display shopping separately from public emissions in footprint charts

### PR DESCRIPTION
**Pull request by stefanlindbohm**
See discussion on Slack.

Screen Shot 2020-11-26 at 10 23 50
Screen Shot 2020-11-26 at 10 16 00

### CODE REVIEW

File: app/views/shared/_chart_categories.html.erb [hunk 1]
```diff
@@ -27,12 +27,18 @@
           title: t('dashboard.footprint.category.flights'),
           'bg-color': 'bg-blue-accent'
         },
+        {
```
Comment (emmabckstrm) [inline app/views/shared/_chart_categories.html.erb:30]: Shouldn't `(@footprint.public.tonnes.round(1) + @footprint.consumption.tonnes.round(1))` be separated here as we are separating the values?

File: app/views/shared/_chart_categories.html.erb [hunk 2]
```diff
@@ -27,12 +27,18 @@
...
+          'bg-color': 'bg-pink-accent'
+        },
         {
           key: 'public',
-          amount: @footprint.public + @footprint.consumption,
+          amount: @footprint.public,
           icon: 'fa-university',
           title: t('dashboard.footprint.category.public'),
-          'bg-color': 'bg-pink-accent'
+          'bg-color': 'bg-gray-accent'
```
Comment (emmabckstrm) [inline app/views/shared/_chart_categories.html.erb:41]: Not really relevant to this specific PR but it makes me think that perhaps we need one more colourful brand colour :D

File: app/views/shared/_chart_categories.html.erb (hunk 1 above)
Comment (emmabckstrm) [reply app/views/shared/_chart_categories.html.erb:30]: At row 3 as I couldn't really comment on that specific row as it's not edited

File: app/views/shared/_chart_categories.html.erb (hunk 2 above)
Comment (stefanlindbohm) [reply app/views/shared/_chart_categories.html.erb:41]: I actually think the gray fit quite well given it's the category you can't change directly. But yeah :D

File: app/views/shared/_chart_categories.html.erb (hunk 1 above)
Comment (stefanlindbohm) [reply app/views/shared/_chart_categories.html.erb:30]: Oh yeah, missed that. Thanks!

**Pull request merged**